ui/app/components/variable-form.js

```diff
@@ -93,7 +95,7 @@ export default class VariableFormComponent extends Component {
     if (!this.args.model?.isNew) {
       keyValues.pushObject(copy(EMPTY_KV));
     }
-    this.keyValues = keyValues;
+    set(this, 'keyValues', keyValues);
```

Comment:
since we're now using this in a computed getter elsewhere, we should use proper `set`ters when establishing our list.